keep/api/models/db/tenant.py

```diff
@@ -37,3 +37,10 @@ class TenantInstallation(SQLModel, table=True):
     bot_id: str
     installed: bool = False
     tenant: Optional[Tenant] = Relationship(back_populates="installations")
+
+
+class TenantAIConfig(SQLModel, table=True):
```

Comment:
? 
why you need a new table for that? use configuration column on the existing tenant table

Reply:
There may be multiple ai configs per tenant, I think we need a separate table for that